db/schema.rb

```diff
@@ -337,7 +337,7 @@
     t.boolean "receive_newsletter"
     t.bigint "source_application_id"
     t.datetime "activated_at"
-    t.boolean "is_newflow", default: false
+    t.boolean "is_newflow", default: false, null: false
```

Comment:
not sure why i'm seeing schema.rb changes again.... hope we can get our migration and schema files in sync w/ master.